### PR TITLE
AAP-18761-A - add assembly for the gateway dashboard

### PR DESCRIPTION
**Pull request by dcdacosta**
This PR adds an assembly file for the gateway dashboard content.

### CODE REVIEW

File: downstream/assemblies/platform/assembly-gw-dashboard.adoc [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+:_mod-docs-content-type: ASSEMBLY
+
+[id="gw-dashboard_{context}"]
+= {PlatformNameShort} dashboard
+
+:context: gw-dashboard
+
+[role="_abstract"]
+The {PlatformName} dashboard provides automation management and monitoring capabilities, allowing you to administer and configure automation functions, as well as view recent job activity details and performance statistics related to it.
```
Comment (sayjadha) [inline downstream/assemblies/platform/assembly-gw-dashboard.adoc:11]: Should this be `{PlatformNameShort}` attribute, just like the title? Ignore this comment if this is intentional.

**Comment by sayjadha** [comment]
One remark/question, rest LGTM.

File: downstream/assemblies/platform/assembly-gw-dashboard.adoc (hunk 1 above)
Comment (dcdacosta) [reply downstream/assemblies/platform/assembly-gw-dashboard.adoc]: Good catch. This is fixed now.

**Review by sayjadha — APPROVED** [review]
LTGM. Merging.

**Pull request merged**